changelog/_unreleased/2021-01-14-add-select-to-property-display-type.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Add select display type to properties
+issue: NEXT-7997
+author: Sebastian Lember, Timo Boomgaarden
+author_email: lember@hochwarth-it.de, boomgaarden@hochwarth-it.de 
+author_github: @sebi007, @timoboomgaarden
+___
```

Comment:
This have to be `---`. Otherwise our changelog parser can´t parse the file :)

Reply:
Just those after author_github or all of them?

Reply:
Only that one